docs/usage/overview.md

```diff
@@ -1,4 +1,229 @@
+they have occurred. Similarly to `databooks meta ...`, the only required argument here
+is a path.
+
+```bash
+databooks fix path/to/notebooks
+```
+
+For each notebook in the path _that has git conflicts_:
+
+- It will keep the metadata from the notebook in [`HEAD`](https://stackoverflow.com/questions/2304087/what-is-head-in-git)
+- For the conflicting cells, it will wrap some special cells around the differences, like
+in normal git conflicts
+
+Similarly to what we saw above, the default behavior can be changed by passing a
+configuration `pyproject.toml` file or specifying the CLI arguments. You could, for
+instance, keep the metadata from the notebook in `BASE` (as opposed to `HEAD`). If you
+know you only care about the notebook cells in `HEAD` or `BASE`, then you could pass
+`--cells-head` or `--no-cells-head` and not worry about fixing conflicted cells in Jupyter.
+
+You can also pass a special `--cell-fields-ignore parameter`, that will remove the cell
+metadata from both versions fo the conflicting notebook before comparing them. This is
+because depending on your Jupyter version you may have an `id` field, that will be unique
+for each cell. That is, all the cells will be considered different even if they have the
+same `source` and `outputs` as their `id`s are different. By removing `id` and
+`execution_count` (we'll do this by default) we only compare the actual code and outputs
+to determine if the cells have changed or not.
+
+!!! note
+
+    If a notebook with conflicts (i.e.: not valid JSON/Jupyter) is committed to the repo,
+    `databooks fix` will not consider the file as something to fix - same behavior as `git`.
+
+!!! info "Fun fact"
+
+    "Fix" may be a misnomer: the "broken" JSON in the notebook is not actually fixed -
+    instead we compare the versions of the notebook that caused the conflict.
+
+## `databooks assert`
+
+In `databooks meta`, we remove unwanted metadata. But sometimes we may still want some
+metadata (such as cell tags), or more than that, we may want the metadata to have
+certain values. This is where `databooks assert` comes in. We can use this command to
+ensure that the metadata is present and has the desired values.
+
+`databooks assert` is akin to (and inspired by) Python's [`assert`](https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement).
+Therefore, the user must pass a path and a string with the expression to be evaluated
+for each notebook.
+
+```bash
+databooks assert path/to/notebooks --expr "python expression to assert on notebooks"
+```
+
+This can be used, for example, to make sure that the notebook cells were executed in
+order. Or that we have `markdown` cells, or to set a maximum number of cells for each
+notebook.
+
+Evidently, there are some limitations to the expressions that a user can pass.
+
+**Variables in scope:**
+
+All the variables in scope are a subclass of Pydantic models. Therefore, you can use them
```

Comment:
This is a lot of stuff to throw at an average data scientist in an overview md IMO. 
Variables in scope? 
Subclass of Pydantic?
The example doesn't make it super clear to a newb like me either 

```suggestion
All of the variables that you may use in your assert expressions are [subclasses of Pydantic models](link). Therefore, you can use **them** 
```
Specify what "them" is here. Is this related to metadata? To cell metadata? Other metadata?

Reply:
That makes a lot of sense! Thanks for the suggestion!